src/pages/HomePage/HomePage.js

```diff
@@ -1,17 +1,15 @@
-import { MuiThemeProvider } from "@material-ui/core";
 import PropTypes from "prop-types";
 import React from "react";
 import HomePageBlurb from "../../components/Content/HomePageBlurb";
 import Header from "../../components/Header/Header";
-import CONSTANTS from "../../helpers/constants";
 
 function HomePage(props) {
     const { startLink, gitHubURL } = props;
     return (
-        <MuiThemeProvider theme={CONSTANTS.THEME()}>
+        <>
```

Comment:
Oh so instead of indicating it as `div` you just leave it blank

Reply:
Reply same as above.